.vscode/settings.json

```diff
@@ -1,3 +1,7 @@
 {
-    "editor.tabSize": 2
-}
\ No newline at end of file
+  "editor.tabSize": 2,
+  "prettier.semi": false,
+  "prettier.singleQuote": true,
+  "prettier.trailingComma": "all",
+  "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
Why wouldn't vscode's prettier know that from our config?

Reply:
mm, it should know the settings, but it doesn't know what the default formatter is.  I'll trim that stuff out we don't need